src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.search-bar__form {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.search-bar__form--big {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  color: #3d4e61;
+  outline: none;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar__input--big {
+  background-position: bottom 51% left 25px;
+  background-size: 19px 19px;
+  padding-left: 62px;
+  height: 66px;
+  font-size: 16px;
+}
+
+.search-bar__input--small {
+  background-position: bottom 51% left 12px;
+  background-size: 11px 11px;
+  padding-left: 33px;
+  height: 38px;
+  font-size: 14px;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar__input:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
The value for the background-color property seems to be incorrect. `linear-gradient` is a value for the `background-image` property, not `background-color`. Please correct this to ensure the intended visual effect is applied.